flink-formats/flink-parquet/src/test/java/org/apache/flink/formats/parquet/row/ParquetRowDataWriterTest.java

```diff
@@ -112,26 +109,27 @@ public class ParquetRowDataWriterTest {
                     TypeConversions.fromLogicalToDataType(ROW_TYPE));
 
     @Test
-    public void testTypes() throws Exception {
+    void testTypes(@TempDir java.nio.file.Path folder) throws Exception {
```

Comment:
nit: Here and in the other test method: We could switch to `File` to avoid having the fully qualified reference.